org-formation/079-access-analyzer/_tasks.yaml

```diff
@@ -0,0 +1,15 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'access_analyzer'
+
+AccessAnalyzer:
+  Type: update-stacks
+  Template: access_analyzer.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}'
+  StackDescription: Setup IAM Access Analyzer service
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: '*'
```

Comment:
I think this is the wrong approach.  The recommended approach is to [enable access analyzer from a delegated admin account](https://docs.aws.amazon.com/IAM/latest/UserGuide/access-analyzer-delegated-administrator.html). 

here's the blog post: https://aws.amazon.com/blogs/aws/new-use-aws-iam-access-analyzer-in-aws-organizations/
```
Today I am pleased to announce that you can create an analyzer in the 
master account or a delegated member account with the entire organization as
the zone of trust.
```

Reply:
@zaro0508 I read the links you shared, which seem to describe how to configure Access Analyzers that function at the *organization* level.  In contrast, this PR create an access analyzer at the *account* level.  Since MS Defender is examining the Synapse Prod account specifically, I believe the PR is correct.

Reply:
Your PR description says.. `activate access analyzer on all accounts`.  Are saying that you only want to enable access analyzer for Synapse Prod account?

Reply:
@zaro0508 That's a good question:  To satisfy MS Defender I think we would only have to active it on the Synapse Prod account, but my thought is to just activate it in all accounts so it's available if needed.

Reply:
Isn't activating for all accounts is basically the same as enabling at the organziation level?  if yes, then the latter is the recommended approach.

Reply:
I believe the `zone of trust` is just a access analyzer configuration.  You can create multiple analyzers with different zone of trust levels.  So I guess the real question is what is your intention for the `zone of trust`?  Is your intention to create an access analyzer in each AWS account with a `zone of trust` set to just that account?   Just be aware that we do have use AWS services that accesses resources across accounts for example: identity center, security hub, guard duty, etc..

Reply:
> I believe the zone of trust is just a access analyzer configuration. You can create multiple analyzers with different zone of trust levels.

You can only create an analyzer with an organization level 'zone of trust' in the organization account (or perhaps in the delegated management account -- I haven't tried that.).

> Is your intention to create an access analyzer in each AWS account with a zone of trust set to just that account?

Yes.